requirements/edx/openstack.txt

```diff
@@ -3,4 +3,4 @@
 #
 
 # OpenStack swift backend for django storage API
-django-storage-swift==1.2.18
+git+https://github.com/open-craft/django-storage-swift.git@jill/content_length_from_fd#egg=django-storage-swift==1.2.19
```

Comment:
@pomegranited I'd suppose edX prefers a hash here, to pin it to a specific commit. After it's merged you can update it.

Reply:
Agreed -- I'm planning to wait for https://github.com/dennisv/django-storage-swift/pull/100 to merge and trigger a version bump, so we can use `django-storage-swift==1.2.19` here.